web/src/styles/base/flex.scss

```diff
@@ -92,6 +92,10 @@
   flex-direction: column-reverse;
 }
 
+.flex-row-reverse {
```

Comment:
[sand] Am i correct in that this is unused?

Reply:
You are indeed correct; this is a fragment left over from a prior implementation that *technically* is no longer needed. However, it's a utility that I'm surprised we didn't have, given that we had its cousin for column-reversing.

Would it be better to leave this in, as someone will undoubtedly use it in the future (and maybe we should consider using something like postCSS.

Reply:
Personally, I'm more in favor of not adding things until we have a need, but that's just me

Reply:
I'm with Anne. If we don't need it, I say kill it (for now).

Reply:
Nuked from orbit. Thanks for the feedback!